package.json

```diff
@@ -3,7 +3,29 @@
   "title": "jQuery",
   "description": "JavaScript library for DOM operations",
   "version": "4.0.0-pre",
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "node": {
+        "module": "./dist/jquery.mjs",
+        "import": "./dist/jquery-node-wrapper.mjs",
+        "default": "./dist/jquery.js"
+      },
+      "browser": "./dist/jquery.mjs"
+    },
+    "./slim": {
+      "node": {
+        "module": "./dist/jquery.slim.mjs",
+        "import": "./dist/jquery-node-wrapper.slim.mjs",
+        "default": "./dist/jquery.slim.js"
+      },
+      "browser": "./dist/jquery.slim.mjs"
+    },
+    "./src/*.js": "./src/*.js",
+    "./amd/*.js": "./amd/*.js"
```

Comment:
These entries are tricky. Technically, we don't guarantee anything about files in those directories, they can come and go even in patch releases. On the other hand, we've been telling people that for advanced cases they can use our modules, taking into account these restrictions.

`src` contains jQuery source written in ES modules, `amd` those modules transpiled to AMD as jQuery 3.x & older used AMD as the source format and we wanted to still make it available to people.

The modules in `src` are compatible with direct usage in a browser without any bundler as well:
```html
<script type="module">
	import jQuery from "./src/jquery.js";
    /* do someting with jQuery */
</script>
```